### ISSUE TITLE
Water under taxiway

### ISSUE THREAD

**Issue by cicku** [openstreetmap/iD#8342]
iD won't alert if water features are across taxiway

![image](https://user-images.githubusercontent.com/686438/106693055-ac510e80-65a3-11eb-95f2-ac3f8e61a058.png)

**Comment by quincylvania** [comment]
Yeah, taxiways and runways are special since they're sort of like roads but aren't tagged that way. Seems reasonable to warn about crossings. See also #7626.

**Comment by tyrasd** [comment]
this was implemented with #9315
